### ISSUE TITLE
[Refactor] Every CrawlWorker has same implementation of `work` function

### ISSUE THREAD

**Issue by timgluz** [versioneye/crawl_r#62]
I noticed that most of `CrawlWorker` are duplicating the `work` method, which could be moved into parent class like the `GosearchVersionWorker` does: https://github.com/versioneye/crawl_r/blob/master/lib/versioneye/workers/crates_crawl_worker.rb

**Comment by reiz** [comment]
Yes. That's a good idea. They can be simplified. But if we do that we have to write some tests for the current implementation. Some of the workers are covered by tests, some not. If we have a 90% test coverage for the existing workers then we can start refactoring. This part is very critical.
